src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -749,25 +753,42 @@ public void addListener(GraphListener listener) {
     public void interrupt(Result result, CauseOfInterruption... causes) throws IOException, InterruptedException {
         setResult(result);
 
+        LOGGER.log(Level.FINE, "Interrupting {0} as {1}", new Object[] {owner, result});
         final FlowInterruptedException ex = new FlowInterruptedException(result,causes);
 
         // stop all ongoing activities
         Futures.addCallback(getCurrentExecutions(/* cf. JENKINS-26148 */true), new FutureCallback<List<StepExecution>>() {
             @Override
             public void onSuccess(List<StepExecution> l) {
+                LOGGER.log(Level.FINE, "interrupt of {0} processed on {1}", new Object[] {owner, l});
                 for (StepExecution e : Iterators.reverse(l)) {
                     try {
                         e.stop(ex);
                     } catch (Exception x) {
-                        LOGGER.log(Level.WARNING, "Failed to abort " + CpsFlowExecution.this.toString(), x);
+                        LOGGER.log(Level.WARNING, "Failed to abort " + owner, x);
                     }
                 }
             }
 
             @Override
             public void onFailure(Throwable t) {
+                LOGGER.log(Level.WARNING, "failed to interrupt steps in " + owner, t);
             }
         });
+
+        // If we are still rehydrating pickles, try to stop that now.
+        Collection<ListenableFuture<?>> futures = pickleFutures;
+        if (futures != null) {
+            LOGGER.log(Level.FINE, "we are still rehydrating pickles in {0}", owner);
```

Comment:
🐜 lowercase again. In this method both lowercase and uppercase letters are being used, so it is a kind of mess